spec/requests/api/drops_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+
+RSpec.describe Api::DropsController, :vcr => {:cassette_name => "place" }, type: :request do
```

Comment:
Thanks for the comments ... can I change the name to anything at all? or what should the name be?